PED/TakeOwnershipOfSynchronizedScene.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: PED
+aliases: ["0xCD9CC7E200A52A6F", "_DISPOSE_SYNCHRONIZED_SCENE"]
+---
+## TAKE_OWNERSHIP_OF_SYNCHRONIZED_SCENE
+
+```c
+// 0xCD9CC7E200A52A6F 0xBF7F9035
+void TAKE_OWNERSHIP_OF_SYNCHRONIZED_SCENE(int scene);
+```
+
+
+## Parameters
+* **scene**: Scene ID returned by (CREATE_SYNCHRONIZED_SCENE)(#_0x8C18E0F9080ADD73)
```

Comment:
Hey, this is wrong, it's supposed to be `[CREATE_SYNCHRONIZED_SCENE](#_0x8C18E0F9080ADD73)`

Thought you should know. 😀

Reply:
Changed it now, is it okay?